### DISCUSSION TITLE
visualizing interleaved QC data

### DISCUSSION THREAD

**Discussion by ycsong** [franciscozorrilla/thesis_pipeline#156] [Q&A] [answered]
Hello,

I was wondering if it is possible to visualize an interleaved QC file using qfilterVis. We have some data that have gone through QC process using BBDuk/BBMap, similar to how JGI does the filtering. 

From reading the tutorial, it appears that qfilterVis works with non-interleaved files (i.e., R1 and R2), unless I am missing something. Thank you in advance.

**Comment by franciscozorrilla** [comment, marked as answer]
Hey Young, sorry I missed this again, for some reason I dont get notified for new discussions.
Please feel free to tag me directly or open an issue so that I am notified in a timely manner.

Regarding your question, indeed the workflow is designed to work with non-interleaved reads, so it expects R1 and R2 files at the qfilter step. However, the qfilterVis rule will go through the additional `.json` statistics files generated by the qfiltering tool `fastp`, and then create a summary file for visualization qfiltering across samples. So if you want to use qfilterVis, you should have fastp output.

Hope this clarifies your question.
Best,
Francisco

**Comment by ycsong** [comment]
Thank you for the clarification, Francisco. Much appreciated.
